clients/webassembly/src/client/config.rs

```diff
@@ -107,6 +107,11 @@ pub struct Debug {
 
 impl From<Debug> for ConfigDebug {
     fn from(debug: Debug) -> Self {
+        // For now we just always use the (older) 32kb extended size
+        let use_extended_packet_size = debug
+            .use_extended_packet_size
+            .then(|| String::from("extended32"));
```

Comment:
IF we're going to go with strings as opposed to enums, `"extended32"` should be exported via some constant in case it was ever changed